big-a/src/_nav.js

```diff
@@ -35,292 +25,39 @@ const _nav = [
     component: CNavItem,
     name: 'Playbook',
     to: '/playbook',
-    icon: <CIcon icon={cilDrop} customClassName="nav-icon" />,
-  },
-  {
-    component: CNavTitle,
-    name: 'Field Validation',
-  },
-  {
-    component: CNavItem,
-    name: 'Field Validation',
-    to: '/fieldvalidation',
-    icon: <CIcon icon={cilCheck} customClassName="nav-icon" />,
+    icon: <CIcon icon={cilNotes} customClassName="nav-icon" />,
+    badge: {
+      color: 'info',
+      text: 'YAML',
+    },
```

Comment:
we can remove this whole section, because it was the "New" highlight textbox but we don't want this feature